### ISSUE TITLE
Health configuration settings

### ISSUE THREAD

**Issue by ericfri** [firehol/netdata#848]
I've just setup netdata and am loving it.  However, the health monitoring settings on one of my servers needed to be tweaked a bit in order to shut it up.  Receiving 100 emails a night is too many.  
After I updated netdata the configuration files were overwritten and thus my changes gone.  
Is there any way to persist my changes through an update?

**Comment by ktsaou** [comment]
They should persist already.

Check `/etc/netdata/health.d`. There should be 3 versions of each file:

- `X.conf` the current one
- `X.conf.old` a backup of the old you had
` `X.conf.orig` the stock version of the file

The installer overwrite `X.conf` with `X.conf.old` only if it is a stock version. It keeps MD5 hashes of all versions of the config files and before copying `X.conf.orig` to `X.conf` it checks you haven't edited it.

I think your edits have not been overwritten. We just added more alarms.

Which alarms bother you? We should make them more strict to prevent sending false alarms.

**Comment by ericfri** [comment]
You are correct.  I checked last night and thought that the .conf files didn't have my saved data, but they seem to.  
The one I see most is the disk fill rate.  I have a nightly backup that runs and it uses my / partition for a tmp storage before moving it off site.  It adds a fair amount of data for a short time (maybe an hour) and then it gets deleted.  
I've bumped the numbers to 96 hours for warn, and 72 for critical and that seems to have helped.
